src/TTextEdit.cpp

```diff
@@ -1655,7 +1655,7 @@ int TTextEdit::bufferScrollUp( int lines )
 
 int TTextEdit::bufferScrollDown( int lines )
 {
-    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()-1 - mScreenHeight) )
+    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()) )
```

Comment:
Yeah, we don't __need__ to force it in here I just thought I'd raise it as I've got yellow warnings all over the place in the IDE (but that is mostly due to implicit variable size/sign-ness castings) - I'm using the following for a (custom) Qt Creator "Tools"->"Options"->"C++"->"Code Model"->"Clang Code Model Warnings":

`-Weverything -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-unused-macros -Wno-newline-eof -Wno-exit-time-destructors -Wno-global-constructors -Wno-gnu-zero-variadic-macro-arguments -Wno-documentation -Wno-shadow -Wno-missing-prototypes -Wno-disabled-macro-expansion -Wno-used-but-marked-unused -ferror-limit=50`

I can mention it in a post to, say: Forum Topic: [Set-Up for Mudlet Developement](http://forums.mudlet.org/viewtopic.php?f=7&t=4619) perhaps?